hls4ml/converters/keras_v2_to_hls.py

```diff
@@ -256,7 +256,6 @@ def parse_keras_model(model_arch, reader):
 
     layer_config = None
     if model_arch['class_name'] == 'Sequential':
-        print('Interpreting Sequential')
```

Comment:
As a general comment for this PR: I personally like that currently hls4ml is giving the user some feedback on what it is actually doing at a given time, especially when running multiple steps in sequence. And printing out the layer structure it creates is also helpful as a sanity check. So I would prefer not to remove all these print outs.